src/WebJobs.Extensions.MobileApps/Bindings/MobileTableQueryValueProvider.cs

```diff
@@ -2,26 +2,25 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Bindings;
 using Microsoft.WindowsAzure.MobileServices;
 
 namespace Microsoft.Azure.WebJobs.Extensions.MobileApps
 {
-    internal class MobileTableQueryValueProvider<T> : IValueProvider
+    internal class MobileTableQueryValueProvider<T> : IValueBinder
```

Comment:
The latest rules should help here and let you skip IValueBinder. 

---
In reply to: [63603726](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63603726) [](ancestors = 63603726,63598844)